packages/react/src/components/DesignLibrary/DLErrorBoundary.tsx

```diff
@@ -0,0 +1,37 @@
+'use client';
+import React, { Suspense } from 'react';
+import { DesignLibraryPreviewError, sendErrorEvent } from '@sitecore-content-sdk/core/codegen';
+
+type DLErrorBoundaryProps = {
+  uid: string;
+  children: React.ReactNode;
+  renderKey?: number;
+};
+
+export class DLErrorBoundary extends React.Component<DLErrorBoundaryProps> {
```

Comment:
Just to continue following our common naming convention
```suggestion
export class DesignLibraryErrorBoundary extends React.Component<DLErrorBoundaryProps> {
```